scripts/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
You should add `set -e` here, or if `terraform apply` fails bash will carry on and it will look like the build succeeded. 

Putting `set -euo pipefail` or similar at the top is a good habit to get into for bash scripts in general.